src/main/java/utils/ReadFileHandler.java

```diff
@@ -0,0 +1,24 @@
+package utils;
+
+import java.io.*;
+import java.util.Properties;
+
+public class ReadFileHandler {
```

Comment:
make it possible to work with different properties, not only with test-data, to make it easier to use if one day you will need to work with more prop files